testdata/ast/compare.in

```diff
@@ -11,3 +11,7 @@ h  not in  j
 l   is m
 
 n  in o
+
+a == b
```

Comment:
This isn't really related to the change. Assuming you revert the one below, I would revert this too.